crates/cubecl-wgpu/src/compute/server.rs

```diff
@@ -127,6 +132,16 @@ impl WgpuServer {
         queue: Arc<wgpu::Queue>,
         tasks_max: usize,
     ) -> Self {
+        let queries = if device.features().contains(wgpu::Features::TIMESTAMP_QUERY) {
+            Some(device.create_query_set(&QuerySetDescriptor {
+                label: Some("CubeCL profile queries"),
+                ty: QueryType::Timestamp,
+                count: 2,
+            }))
+        } else {
+            None
+        };
+
```

Comment:
A quick feedback, the fix in this PR now works on Github Actions, locally on M1 too. Thanks!